evil-surround.el

```diff
@@ -65,8 +65,11 @@ Each item is of the form (TRIGGER . (LEFT . RIGHT)), all strings.
 Alternatively, a function can be put in place of (LEFT . RIGHT).
 This only affects inserting pairs, not deleting or changing them."
   :group 'surround
-  :type '(repeat (cons (regexp :tag "Key")
-                       (symbol :tag "Surround pair"))))
+  :type '(alist
```

Comment:
see https://www.gnu.org/software/emacs/manual/html_node/elisp/Association-Lists.html

this `defcustom` uses dotted pairs, and these `:type` annotations tell the emacs customize interface how to build a ui for this configuration item; see screenshot.

please try it out before and after the change: `M-x customize-variable` then choose the variable `evil-surround-pairs-alist`.